### ISSUE TITLE
Normalize time units between data and MC

### ISSUE THREAD

**Issue by cvilelahep** [SND-LHC/sndsw#246]
Currently the unit for the hit time returned by the hit classes is different for data (clock cycles) and MC (ns). This generates confusion. It would be better to change the code such that times are always returned in the same unit.

**Comment by olantwin** [comment]
I agree, we should stick to one unit, with exceptions where it's unavoidable for e.g. performance reasons or size in memory, which then should be clearly documented.

I'd probably prefer aligning to ns by default.

**Comment by cvilelahep** [comment]
For future reference, if this gets implemented, undo #264

**Comment by GiuliaPaggi** [comment]
This should also be accounted in sndScifiPlane.* and sndUSPlane.*
